cg/meta/demultiplex/utils.py

```diff
@@ -206,11 +207,18 @@ def parse_manifest_file(manifest_file: Path) -> list[Path]:
 
 def is_file_relevant_for_demultiplexing(file: Path) -> bool:
     """Returns whether a file is relevant for demultiplexing."""
-    relevant_directories = [DemultiplexingDirsAndFiles.INTER_OP, DemultiplexingDirsAndFiles.DATA]
-    for relevant_directory in relevant_directories:
-        if relevant_directory in file.parts:
-            return True
-    return False
+    irrelevant_directories = [
+        "Thumbnail_Images",
+        ".tmp.cbcl",
+        ".cbcl.tmp",
+        "MyRun",
+        "tmp.filter",
+        "CopyComplete.txt",
```

Comment:
Not sure if these should be constants, Since we per definition fin them irrelevant... 😅

Reply:
No complaints from peers. Keeping as is